sapl/base/models.py

```diff
@@ -40,6 +40,11 @@
     ('T', _('Todos os Parlamentares Presentes na Sessão')),
 )
 
+ORDENACAO_PESQUISA_MATERIA = (
+    ('A', _('Alfabetica pela Sigla')),
```

Comment:
```suggestion
    ('A', _('Alfabética por sigla')),
```

Nit: poderia trocar o caracter de 'A' pra 'S', por favor?